features/step_definitions/lesson_steps.rb

```diff
@@ -61,6 +62,18 @@
   end
 end
 
+When /^I mark all the lessons as completed$/ do
+  @lessons.each do |lesson|
+    step "I mark the lesson '#{lesson.title}'"
+  end
+end
+
+When /^I go back$/ do
+  # The following doesn't work
+  # page.execute_script('window.history.back()')
```

Comment:
I'm not really sure, I haven't seen anything like that before. I think what you have is better though it makes it very clear and explicit about going to the courses page. Remember clear and readable beats clever and hard to understand code every time ;) so the simplest solution is usually the best.